libs/article/website/src/lib/article-container.tsx

```diff
@@ -24,5 +30,24 @@ export function ArticleContainer({onQuery, id, slug, className}: ArticleContaine
     onQuery?.({data, loading, error, refetch})
   }, [data, loading, error, refetch, onQuery])
 
-  return <Article data={data} loading={loading} error={error} className={className} />
+  return (
+    <PollBlockProvider>
+      <Article data={data} loading={loading} error={error} className={className} />
+    </PollBlockProvider>
+  )
+}
+
+function PollBlockProvider({children}: PropsWithChildren) {
```

Comment:
prob. not a big deal, but it's defined after using

Reply:
Hoisting does the trick because it's used inside a function :D